src/pilot/assign_role.py

```diff
@@ -373,6 +382,18 @@ def get_raid_controller_physical_disk_ids(drac_client, raid_controller_fqdd):
         key=physical_disk_id_to_key)
 
 
+def is_boss_controller(raid_disks, raid_ctrl_name, drac):
+    if is_raid_controller(raid_ctrl_name, drac):
```

Comment:
This method should just return True if the controller is a BOSS and False if not.

Reply:
Got it.